plugin-flex-ts-template-v2/src/feature-library/teams-view-columns/index.ts

```diff
@@ -0,0 +1,7 @@
+import { FeatureDefinition } from '../../types/feature-loader';
+// @ts-ignore
+import hooks from './flex-hooks/**/*.*';
```

Comment:
functionally this is fantastic but its missing some of the boilerplate.  It looks like you probably didnt add this feature using "npm run add-feature teams-view-columns".  This would have created some of the boiler plate that is missing.  Namely the config and the types so we can control whether this feature is enabled or not.

if you try running the script now it will abandon because the feature folder is already there.  I might suggest we create a new feature and call it "teams-view-enhancements" where in the future we can load in a suite of different configurable enhancements but for now just have the columns changes.

to do this

- run the script `npm run add-feature teams-view-enhancements`
- copy your code into the boilerplate
- remove the teams-view-columns folder
- add in a docs page for the feature

Reply:
Done